pkg/config/config.go

```diff
@@ -105,20 +107,22 @@ var (
 	ModelServerEnable   = getBoolConfig("MODEL_SERVER_ENABLE", false)
 	ModelServerEndpoint = SetModelServerReqEndpoint()
 	// for model config
-	modelConfigValues map[string]string
-	// model_item
-	NodeTotalKey           = "NODE_TOTAL"
-	NodeComponentsKey      = "NODE_COMPONENTS"
-	ContainerTotalKey      = "CONTAINER_TOTAL"
-	ContainerComponentsKey = "CONTAINER_COMPONENTS"
-	ProcessTotalKey        = "PROCESS_TOTAL"
-	ProcessComponentsKey   = "PROCESS_COMPONENTS"
-
-	//  attribute
-	EstimatorEnabledKey = "ESTIMATOR"
-	InitModelURLKey     = "INIT_URL"
-	FixedModelNameKey   = "MODEL"
-	ModelFiltersKey     = "FILTERS"
+	ModelConfigValues map[string]string
+	// model_parameter_prefix
+	NodePlatformPowerKey        = "NODE_TOTAL"
```

Comment:
We may have to change later but for now "TOTAL" = "acpi" source while "COMPONENTS" = "rapl" source.